cc/verify.go

```diff
@@ -0,0 +1,58 @@
+package cc
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"math/big"
+
+	"github.com/anoideaopen/acl/helpers"
+	aclproto "github.com/anoideaopen/acl/proto"
+	pb "github.com/anoideaopen/foundation/proto"
+	"github.com/btcsuite/btcutil/base58"
+	"golang.org/x/crypto/ed25519"
```

Comment:
can use this package `crypto/ed25519`? 
In which go it is added to the baseline? 
We're still at go1.18.